docs/extensibility/extend-rest-api-formatters.md

```diff
@@ -0,0 +1,95 @@
+# Formatters
+As part of ExtendRestAPI, we introduced `Formatters`, these are utility classes that allow you to format values to
+so that they are compatible with the StoreAPI, values such as money, currency, or HTML.
+
+## How to use them
+To get a formatter, you can use the `get_formatter` method of the `ExtendRestApi` class. This method accepts a string,
+which is the name of the formatter you want to use, e.g. (money, html, currency).
```

Comment:
It would be helpful to summarize this `This method accepts a string,
which is the name of the formatter you want to use, e.g. (money, html, currency).` just by adding or replacing it with a code example. 
```
get_formatter('money'); // For the MoneyFormatter
get_formatter('html'); // For the HtmlFormatter
get_formatter('currency'); // CurrencyFormatter
```